stable/drupal/meta.yaml

```diff
@@ -1,5 +1,5 @@
 name: "drupal"
-description: "drupal is a free and open-source content management system (CMS) for publishing web content. It is built on a model–view–controller web application framework that can be used independently of the CMS."
+description: "drupal is a free and open-source content management system CMS for publishing web content. It is built on a model–view–controller web application framework that can be used independently of the CMS."
```

Comment:
I'm newbie in English, but is it ok to use "is" as auxiliary verb of the present simple tense and "built" verb in the past tense? Shouldn't it be replaced to "was built"?

Reply:
Here, "built" means "made with". I would keep the " It is built on".